src/coreclr/tests/issues.targets

```diff
@@ -1687,6 +1700,183 @@
         <ExcludeList Include="$(XunitTestBinBase)/Interop/PInvoke/Generics/GenericsTest/**">
             <Issue>needs triage</Issue>
         </ExcludeList>
+	<ExcludeList Include = "$(XunitTestBinBase)/Interop/DllImportAttribute/ExactSpelling/ExactSpellingTest/**">
```

Comment:
Nit - please try to keep indentation consistent in the file for easier orientation.